.github/ISSUE_TEMPLATE/issue--bug-report.md

```diff
@@ -0,0 +1,97 @@
+---
+name: 'Issue: Bug report'
+about: Let us know about an unexpected error or incorrect behavior
+title: "[BUG]"
+labels: bug
+assignees: ''
+
+---
+
+<!--
+Welcome,
+
+Thanks for opening an issue. The more information you provide, the easier it is for us to assess the problem, prioritize, assign, develop, then release a fix.
+
+The IAC/Deployment team
+
+-->
+
+### Terraform Version
+
+<!---
+We ask this to be sure you are currently running a supported terraform version from your work environment. 
+
+Run `terraform version` to show the version, and paste the result between the ``` marks below.
+
+If you are not running the latest version of Terraform we support, please try upgrading because your issue may have already been fixed.
+
+If you're not sure which versions are supported, here's a link : https://github.com/sassoftware/viya4-iac-azure#terraform to help.
+-->
+
+```bash
+...
+```
+
+### Terraform Configuration Files
+
+<!--
+Paste the relevant parts of your Terraform configuration between the ``` marks below.
+
+Configuration files you write in Terraform language tell Terraform what plugins to install, what infrastructure to create, and what data to fetch.
+These are the `*.tf` files in this directory.
+
+The relevant parts should come from your `terraform.tfvars` file or equivalent and small snippets of the `*.tf` file/files that seem to be causing the error.
+
+For Terraform config files larger than a few resources, provide a link to the file/files in a current repository, rather than copy-pasting multiple files in here.
+
+For security reasons, do not copy and paste any sensitive material in this issue.
```

Comment:
replace 'material' with 'information', like account information and passwords etc.

Reply:
done